saleor/graphql/shop/schema.py

```diff
@@ -27,7 +29,14 @@ class ShopQueries(graphene.ObjectType):
     )
     order_settings = PermissionsField(
         OrderSettings,
-        description="Order related settings from site settings.",
+        description=(
+            "Order related settings from site settings."
+            "\n Returned `orderSettings` will be first `channel` in "
+            "alphabetical order."
+        ),
+        deprecation_reason=(
+            f"{DEPRECATED_IN_3X_FIELD} " "Use the `channel` field instead."
```

Comment:
f"{DEPRECATED_IN_3X_FIELD} Use the `channel` field instead."
No need two `"` inside the string.
Also it should be something like. 
Use the `channel` query to fetch the `orderSettings` field instead.